spec/factories/transient_registration.rb

```diff
@@ -87,8 +93,25 @@
 
     trait :has_revoked_registration do
       # Create a new registration when initializing so we can copy its data
-      initialize_with { new(reg_identifier: create(:registration, :has_required_data,
-      metaData: build(:metaData, revoked_reason: "foo")).reg_identifier) }
+      initialize_with do
```

Comment:
I think we can make more use of some of the existing registration factory traits for these new transient_registration traits (like `:is_revoked`).

I also wonder if it'd be better to have new registration traits like `:expired_one_month_ago` to do the work rather than specifying details in these `initialize_with` blocks, as I think this would be more consistent with other traits in this factory.

Reply:
Good point, though I think we have a general mix of using before/let blocks, or using Factorybot throughout the tests.

Would you be happy that to get this into production (as we have users with stuck registrations waiting on this)

- Where we have an existing trait I update the tests to use it
- Where we don't we leave as is and add an issue to review our factories and the traits required

Reply:
Sounds good 👍

Reply:
Have updated the code and also created issue #322